init.lua

```diff
@@ -147,18 +144,19 @@ function ranks.update_privs(player, trigger)
 end
 
 -- [function] Update player nametag
-function ranks.update_nametag(player)
+function ranks.update_nametag(name)
 	if minetest.settings:get("ranks.prefix_nametag") == "false" then
 		return
 	end
 
-	if type(player) == "string" then
-		player = minetest.get_player_by_name(player)
+	if type(name) ~= "string" then
+		player = name:get_player_name()
+	else
+		player = minetest.get_player_by_name(name)
```

Comment:
This should be replaced with
```lua
if type(name) ~= "string" then
    name = name:get_player_name()
end
```
as `player` should be defined on line 169 immediately before the if statement where it is necessary.

Reply:
![image](https://user-images.githubusercontent.com/49789044/103149346-75940700-4760-11eb-8157-e54c92068f5a.png)

Like so?

Reply:
Ah, I have seen your next review. Changed it to:
`local player = minetest.get_player_by_name(name)`